src/components/test/UserAvatar.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { shallow } from "enzyme";
+import UserAvatar from "../UserAvatar";
+import { getPrefabProfile } from "../../test/testProfiles";
+
+describe("UserAvatar", () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
```

Comment:
I don't think you have any mocks here... yet.

Reply:
I think we can remove the resetAllMocks here since we have no mocks.